tutorials/fundamentals/compare-without-time.md

```diff
@@ -0,0 +1,30 @@
+If you want to compare two dates in JavaScript without using the time aspect, you should use
+the `toDateString()` method. It returns the date portion of the `Date` object as a string. From there,
+you can compare the two strings:
+
+```javascript
+const date1 = new Date('2000-06-25');
+const date2 = new Date('2000-06-25');
+
+date1 === date2 // false
+date1.toDateString() === date2.toDateString() // true
+```
+
+## Handling Time Zones
+If you want to determine if two dates are on the same day in a specific timezone, you should use `toLocaleDateString()` for better timezone support. Make
+sure to specify the `locales` and `options` argument, as the default is implementation dependent.
```

Comment:
No need to specify `locales`, just indicate that you need to set the `timeZone` option to tell JavaScript what time zone to use when converting the date to a string.